python/create_client/__init__.py

```diff
@@ -9,3 +9,48 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 from create_client import *
```

Comment:
Looks like the only thing imported here is `CreateClient`. That being the case, mind ditching the * import? That'll appease hound and also make it more obvious where the `CreateClient` class is coming from.

Reply:
+1 to that. Using `import *` is very dangerous. For example, it will prevent a tool like `pylint` or `pyflake8` to detect if you're code is using a method or variable that doesn't exist in the current scope.

Reply:
It doesn't help that our code shows up as a Christmas tree in flake8 either, as it drowns these errors in a sea of whitespace warnings, but at least we're resolving that as we port to Python 3.